staging/src/k8s.io/sample-controller/artifacts/examples/crd.yaml

```diff
@@ -1,10 +1,33 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: foos.samplecontroller.k8s.io
+  # for more information on the below annotation, please see
+  # https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/2337-k8s.io-group-protection/README.md
+  annotations:
+    "api-approved.kubernetes.io": "unapproved, experimental-only; please get an approval from Kubernetes API reviewers if you're trying to develop a CRD in the *.k8s.io or *.kubernetes.io groups"
 spec:
   group: samplecontroller.k8s.io
-  version: v1alpha1
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+        # schema used for validation
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                deploymentName:
+                  type: string
+                image:
```

Comment:
`image` isn't part of the [API](https://github.com/kubernetes/kubernetes/blob/22d120b4d68c8993f2782b9020b230e15f626c3c/staging/src/k8s.io/sample-controller/pkg/apis/samplecontroller/v1alpha1/types.go#L35-L39). Let's remove it